c_glib/arrow-glib/composite-array.cpp

```diff
@@ -90,16 +90,17 @@ garrow_list_array_class_init(GArrowListArrayClass *klass)
  * Since: 0.4.0
  */
 GArrowListArray *
-garrow_list_array_new(gint64 length,
+garrow_list_array_new(GArrowDataType *data_type,
```

Comment:
We need to update parameter list in comment.
We need to add `@data_type: ...`.

Reply:
Added.